plugins/cilium-cni/main.go

```diff
@@ -547,7 +554,7 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 	}
 
 	if ipv4IsEnabled(ipam) {
-		ep.Addressing.IPV4 = ipam.Address.IPV4
+		ep.Addressing.IPV4 = ipam.IPV4.IP
```

Comment:
Why is this change necessary?

Reply:
My bad this is not necessary. I was testing something. Let me remove this.